interactions/client/bot.py

```diff
@@ -1416,6 +1421,33 @@ async def modify(
 
         return User(**data)
 
+    async def get(self, obj: Type[_T], **kwargs) -> _T:
+        """
+        A helper method for retrieving data from the Discord API in its object representation.
+
+        :param obj: The object to get. Should be a class object (not an instance!). For example: `interactions.Channel`.
+        :type obj: object
+        :param **kwargs: The arguments to pass to the HTTP method.
+        :type **kwargs: dict
+        :return: The object we're trying to get.
+        :rtype: object
+        """
+
+        if isinstance(obj, (Channel, Emoji, Guild, Member, Message, Role, Sticker, User)):
```

Comment:
```suggestion
        if isinstance(obj, type):
```

Reply:
This checks if something is a class and not an instance [here is link](https://stackoverflow.com/questions/18499614/check-if-class-object)

Reply:
That would do the opposite effect, where passing a class would raise the error instead of when an instance is passed. I do think that not hardcoding objects would be beneficial, though `inspect.isclass` would probably be the best way to do it

Reply:
Oops forgot to add `not`